deploy/alerts-preprod.json

```diff
@@ -0,0 +1,29 @@
+{
+    "apiVersion": "nais.io/v1alpha1",
+    "kind": "Alert",
+    "metadata": {
+        "name": "speil-alerts",
+        "labels": {
+            "team": "tbd"
+        }
+    },
+    "spec": {
+        "receivers": {
+            "slack": {
+                "channel": "#speil",
```

Comment:
Denne kanalen skulle vel legges ned, og @davidsteinsland skulle opprette en egen kanal for alerts - er den på plass?

Reply:
Den kom på plass for få minutter siden, endrer